pkg/apis/addtoscheme_app_v1alpha1.go

```diff
@@ -15,29 +15,22 @@
 package apis
 
 import (
-	monv1 "github.com/coreos/prometheus-operator/pkg/apis/monitoring/v1"
-	infinispanv1 "github.com/infinispan/infinispan-operator/pkg/apis/infinispan/v1"
-	keycloakv1alpha1 "github.com/keycloak/keycloak-operator/pkg/apis/keycloak/v1alpha1"
 	"github.com/kiegroup/kogito-cloud-operator/pkg/apis/app/v1alpha1"
-	kafkabetav1 "github.com/kiegroup/kogito-cloud-operator/pkg/apis/kafka/v1beta1"
-	oappsv1 "github.com/openshift/api/apps/v1"
-	buildv1 "github.com/openshift/api/build/v1"
-	oimagev1 "github.com/openshift/api/image/v1"
-	routev1 "github.com/openshift/api/route/v1"
-	appsv1 "k8s.io/api/apps/v1"
 )
 
 func init() {
 	// Register the types with the Scheme so the components can map objects to GroupVersionKinds and back
-	AddToSchemes = append(AddToSchemes,
-		v1alpha1.SchemeBuilder.AddToScheme,
+	AddToSchemes = append(AddToSchemes, v1alpha1.SchemeBuilder.AddToScheme)
+	/*
 		kafkabetav1.SchemeBuilder.AddToScheme,
-		oappsv1.Install,
-		routev1.Install,
-		oimagev1.Install,
-		buildv1.Install,
-		appsv1.AddToScheme,
-		monv1.AddToScheme,
-		infinispanv1.AddToScheme,
-		keycloakv1alpha1.SchemeBuilder.AddToScheme)
+			oappsv1.Install,
+			routev1.Install,
+			oimagev1.Install,
+			buildv1.Install,
+			appsv1.AddToScheme,
+			monv1.AddToScheme,
+			infinispanv1.AddToScheme,
+			keycloakv1alpha1.SchemeBuilder.AddToScheme
+
+	*/
```

Comment:
Why removing these? How does it work instead?

Reply:
LOL, I have to remove the comments. We don't need this anymore since now we handle the Schema with the `dependency_management.go`. Please take a look on it.